backends/edk2-compat/edk2-svc-generate.c

```diff
@@ -16,15 +16,24 @@
 
 
 struct Arguments {
-    // the alreadySignedFlag is to determine if signKeys stores a private key file(0) or signed data (1)
-	int helpFlag, inpValid, signKeyCount, signCertCount, alreadySignedFlag;
+    // the pkcs7_gen_meth is to determine if signKeys stores a private key file(0) or signed data (1)
+	int helpFlag, inpValid, signKeyCount, signCertCount, pkcs7_gen_meth;
```

Comment:
should pkcs7_gen_meth have type `enum pkcs7_generation_method`?

Reply:
Yes... Whoops